modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -82,6 +103,165 @@ export async function createCombinedKey(
   };
 }
 
+/**
+ * Creates the SignShare with User XShare and YShare Corresponding to BitGo
+ * @param {XShare} xShare User secret xShare
+ * @param {YShare} yShare YShare from Bitgo
+ * @returns {Promise<SignShare>}
+ */
+export async function createUserSignShare(xShare: XShare, yShare: YShare): Promise<SignShare> {
+  if (xShare.i === ShareKeyPosition.BITGO) {
+    throw new Error(`Invalid XShare, XShare doesn't belong to the User`);
+  }
+
+  if (yShare.i === ShareKeyPosition.BITGO || yShare.j !== ShareKeyPosition.BITGO) {
```

Comment:
`if (yShare.i === ShareKeyPosition.BITGO || yShare.j === ShareKeyPosition.User) {`

Reply:
I didn't explicitly mention user because backup could also be used to sign as the second part instead of user share, although could be only user share since backup would probably be only used in ovc, want me to restrict only to user?

Reply:
ah good point